src/Traits/GraphQLModelTrait.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Nuwave\Relay\Traits;
+
+use GraphQL\Type\Definition\ResolveInfo;
+use Illuminate\Database\Eloquent\Builder;
+
+trait GraphQLModelTrait
```

Comment:
Let's remove this from this PR since you have another one that takes care of this trait.